pkg/dataobj/tools/stats.go

```diff
@@ -24,6 +26,12 @@ type SectionsPerTenantStats struct {
 	P99    float64
 }
 
+type SectionSizeStats struct {
+	Median float64
+	P95    float64
+	P99    float64
+}
+
```

Comment:
How is this struct different to `SectionsPerTenantStats`? Couldn't the existing be reused?

Reply:
I renamed the struct to `PercentileStats`, I think that's better?